app/build.gradle

```diff
@@ -103,7 +103,9 @@ dependencies {
     implementation "com.google.android.gms:play-services-base:$firebaseVersion"
     implementation "com.google.firebase:firebase-messaging:$firebaseVersion"
     implementation "com.google.android.gms:play-services-location:$firebaseVersion"
-    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.2'
+    implementation('com.crashlytics.sdk.android:crashlytics:2.9.3@aar') {
+        transitive = true
```

Comment:
Do we really need this? We don't have library projects?

Instead we should use this:
```
android {
    buildTypes {
        debug {
          // Disable fabric build ID generation for debug builds
          ext.enableCrashlytics = false
```

Reply:
Go ahead and try it.
`ext.enableCrashlytics = false` immediately crashes the app, before any of our code can run...